lib/managers/KalturaTrackingManager.js

```diff
@@ -0,0 +1,145 @@
+const util = require('util');
+const udpsender = require('../utils/udpsender');
+const crypto = require('crypto');
+const kaltura = require('../KalturaManager');
+
+/**
+ * @service tracking
+ */
+class KalturaTrackingManager extends kaltura.KalturaManager
+{
+    constructor()
+    {
+        super();
+    }
+
+    /***
+     * create a url to call the sendBeacon action when beacon tracking is needed.
+     */
+    static generateBeaconRequest(domain, partnerId, params)

```

Comment:
why do you get the domain , it automatically takes it from the configuration (should be fine for testing as well) as the testing machine will have its own configuration files